convert-logos.sh

```
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Script to convert logo PNGs to WebP format
+# Run this after installing webp tools: brew install webp
+
+echo "Converting logo PNGs to WebP..."
+
+cd public/logo
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)